game_loader.py

```diff
@@ -0,0 +1,91 @@
+from email.policy import default
+from importlib.resources import path
+from operator import le
+from operator import mul
+import os
+import textwrap
+import click
+from pyfiglet import Figlet
+from crpg import generate
+from functools import reduce
+from os.path import exists
+
+#Allows adding specifications when starting program; at the moment, only add the abitly to specify exact file path for .dl
+@click.command()
+@click.option("--path", default="game_libary/", help="Provide file path to a .dl file not located under /game_libary")
```

Comment:
This is perfect! It works exactly as I would want to set the correct path and read from it